tests/testProjectedLikelihood.py

```diff
@@ -191,8 +192,7 @@ def testDirect(self):
         ctrl = lsst.meas.modelfit.UnitTransformedLikelihoodControl()
         var = numpy.random.rand(self.bbox0.getHeight(), self.bbox0.getWidth()) + 2.0
         self.exposure0.getMaskedImage().getVariance().getArray()[:, :] = var
-        efv = lsst.meas.modelfit.EpochFootprintVector()
-        efv.push_back(lsst.meas.modelfit.EpochFootprint(self.footprint0, self.exposure0, self.psf0))
+        efv = [lsst.meas.modelfit.EpochFootprint(self.footprint0, self.exposure0, self.psf0)]
```

Comment:
I'm worried about the performance penalties we *can't* easily work around with ndarray (because the item type isn't a numpy type).  I haven't seen any cases that worry me.

And in fairness to Swig, I probably could have used the same syntax for these lines back in the Swig days, and if I wasn't it was because I defined the vector wrappers and the wrappers for this function in the wrong order.  But I do agree with you in general.